raiden/messages/encode.py

```diff
@@ -22,36 +22,24 @@
     SendSecretReveal,
     SendUnlock,
 )
-from raiden.utils.typing import MYPY_ANNOTATION
+
+
+EVENT_MAP = {
```

Comment:
Please make this private (by prepending and underscore) if it does not need to be public, to discourage usage outside of this module.